api/src/main/java/com/kodemon/api/dto/UserDTO.java

```diff
@@ -19,6 +19,10 @@
     private String lastName;
     @JsonProperty(required = true)
     private Date dateOfBirth;
+    @JsonProperty(required = true)
```

Comment:
Definitely not. That would allow anybody set himself as admin. I would rather use `@JsonIgnore` annotation to forbid users to set this property at all. Moreover, please set the default value of this field to false. I think that promoting users to the "admin" role should be done just by another admin, so we can consider adding method `promoteToAdmin()` or sth like that to the user facade.